Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -233,6 +233,12 @@ private void OnRunLevelChanged(GameRunLevelChangedEvent ev)
         }
     }
 
+    public void OnLoneOpsSpawn()
```

Comment:
Name this something else, because this name implies its an event handler, which it isn't.